temporal/api/cloud/cloudservice/v1/service.proto

```diff
@@ -125,4 +125,49 @@ service CloudService {
             get: "/api/v1/cloud/regions/{region}",
         };
     }
+
+    // Get all groups
+    rpc GetGroups (GetGroupsRequest) returns (GetGroupsResponse) {
+        option (google.api.http) = {
+            get: "/api/v1/groups",
+        };
+    }
+
+    // Get a group
+    rpc GetGroup (GetGroupRequest) returns (GetGroupResponse) {
+        option (google.api.http) = {
+            get: "/api/v1/groups/{group_id}",
+        };
+    }
+
+    // Create new a group
+    rpc CreateGroup (CreateGroupRequest) returns (CreateGroupResponse) {
+        option (google.api.http) = {
+            post: "/api/v1/groups",
+            body: "*"
+        };
+    }
+
+    // Update a group
+    rpc UpdateGroup (UpdateGroupRequest) returns (UpdateGroupResponse) {
+        option (google.api.http) = {
+            post: "/api/v1/groups/{group_id}",
```

Comment:
It is a bit confusing that I can't POST with my ID to create the thing even though it is a user-supplied ID. Consider combining create and update into one operation using resource version to differentiate.